src/api-client.ts

```diff
@@ -20,87 +14,63 @@ const enum ContentTypes {
     SRT = 'application/x-subrip'
 }
 
+// tslint:disable-next-line
+const sdkVersion = require('../package.json').version;
+
 export class RevAiApiClient {
     accessToken: string;
-    version: string;
-    instance: AxiosInstance;
+    apiHandler: ApiRequestHandler;
     constructor (accessToken: string, version = 'v1') {
+        this.apiHandler = new ApiRequestHandler(`https://api.rev.ai/revspeech/${version}/`, {
+            'Authorization': `Bearer ${accessToken}`,
+            'User-Agent': `RevAi-NodeSDK/${sdkVersion}`
```

Comment:
The User Agent header does not need to be defined here. We could define it within ApiRequestHandler then we wouldn't have to pass it through the constructor.


Same with the `https://api.rev.ai/revspeech/` part of `https://api.rev.ai/revspeech/${version}/`.

I think we'll only need to pass the version number.

I would refactor the constructor of ApiRequestHandler to `ApiRequestHandler(accessToken: string, version: string)`